src/jdk.jlink/share/classes/jdk/tools/jlink/resources/jlink.properties

```
@@ -111,10 +111,16 @@ main.extended.help.footer=\
 \              used, one pattern per line\n\
 \n\
 
+main.capability.runtime=Capabilities: {0}run-time-image
 
 error.prefix=Error:
 warn.prefix=Warning:
 
+err.runtime.link.not.linkable.runtime=This JDK does not support linking from the current run-time image
+err.runtime.link.jdk.jlink.prohibited=This JDK does not contain packaged modules\
+\ and cannot be used to create another image with the jdk.jlink module
+err.runtime.link.packaged.mods=--keep-packaged-modules is not allowed. This run-time image capable JDK\
+\ does not include packaged modules
```

I think this error indicates that this JDK does not have any packaged modules, i.e. no default module path `jmods` directory.

It may be clearer to say:

```suggestion
err.runtime.link.packaged.mods=This JDK has no packaged modules.  --keep-packaged-modules is not supported.
```

---

Thanks!